deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -211,48 +211,18 @@ def _execute_compaction(
 
         logger.info(f"Round completion file: {round_completion_info}")
 
+    # discover deltas
     delta_discovery_start = time.monotonic()
-
-    input_deltas: List[Delta] = io.discover_deltas(
-        params.source_partition_locator,
-        params.last_stream_position_to_compact,
-        params.rebase_source_partition_locator,
-        params.rebase_source_partition_high_watermark,
-        high_watermark,
-        params.deltacat_storage,
-        params.deltacat_storage_kwargs,
-        params.list_deltas_kwargs,
-    )
-    if not input_deltas:
+    (
+        uniform_deltas,
+        delete_strategy,
+        delete_file_envelopes,
+        delete_file_size_bytes,
+        no_input_deltas,
```

Comment:
`no_input_deltas` arg is redundant as you can determine that from `uniform_deltas`.

Reply:
Removed this argument. I started by checking `if not uniform_deltas` but the pytests failed. Seems like there are cases where there are `input_deltas` and `uniform_deltas` is None. Currently I return `uniform_deltas = False` and check `if uniform_deltas == False`, and the pytests pass. The linter seems to be complaining about this and suggests going back to `if not uniform_deltas` instead. Not sure if there is a better way to do this?

Reply:
The latter? uniform_deltas is not a boolean so comparing it with boolean doesn't make sense.